packages/clerk-js/src/core/clerk.redirects.test.ts

```diff
@@ -128,18 +128,30 @@ describe('Clerk singleton - Redirects', () => {
 
       it('redirects to signInUrl', async () => {
         await clerkForProductionInstance.redirectToSignIn({ redirectUrl: 'https://www.example.com/' });
-        expect(mockNavigate).toHaveBeenNthCalledWith(1, '/sign-in#/?redirect_url=https%3A%2F%2Fwww.example.com%2F');
+        expect(mockNavigate).toHaveBeenNthCalledWith(
+          1,
+          '/sign-in#/?after_sign_in_url=%2F&after_sign_up_url=%2F&redirect_url=https%3A%2F%2Fwww.example.com%2F',
```

Comment:
I guess this is the correct result, cause it overrides the AP values with the new `/` default. My only concern is if this is considered too much noise on the address bar. But I don't think there is another way to do this without changing the AP values.

cc @dimkl @nikosdouvlis WDYT?

Reply:
Yeah that was my concern to that maybe this will make to much noise, but I couldn't find a better way to do it without doing a major refactor on how do we build URLs.

Reply:
I think that we should use the `/` as a fallback in the code instead of passing them as parameters in the URL.

Reply:
@dimkl Done!